modules/microshift-mirroring-prereqs.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: CONCEPT
+[id="microshift-configuring-mirroring-prereqs_{context}"]
+= Configuring mirroring prerequisites
+
+You must create a container image registry credentials file that allows the mirroring of images from your Internet-connected mirror host to your air-gapped mirror. Follow the instructions in the "Configuring credentials that allow images to be mirrored" link provided in the Additional resources to create a `~/.pull-secret-mirror.json` file on the mirror registry host that includes the user credentials for accessing the mirror.
+
+[id="microshift-example-mirror-pull-secret-entry"]
+== Example mirror registry pull secret entry
+
+For example, the following section is added to the pull secret file for the `microshift-quay:8443` mirror registry using `microshift:microshift` as username and password.
+
+.Example mirror registry section for pull secret file
+[source,terminal]
+----
+"<microshift_quay:8443>": { <1>
+    "auth": "<microshift_auth>", <2>
+    "email": "<microshift_quay@example.com>" <3>
+},
+----
+<1> Replace <microshift-quay:8443> with the host name and port of your mirror registry server.
+<2> Replace <microshift_auth> with the user password.
+<3> Replace <microshift_quay@example.com> with the user email.
```

Comment:
```suggestion
<1> Replace `<microshift-quay:8443>` with the host name and port of your mirror registry server.
<2> Replace `<microshift_auth>` with the user password.
<3> Replace `<microshift_quay@example.com>` with the user email.
```
Not sure exactly, but also might need to escape the email address with a `\` so that it doesn't become hyperlinked/clickable.